tests/unit/app/validations/read_grow_test.py

```diff
@@ -1,11 +1,22 @@
 from app.validation.read_grow import validate_read_grow
-from unittest.mock import MagicMock
+from unittest.mock import MagicMock, Mock
 import pytest
 
+
 def test_read_grow_fail():
     AppConfig = MagicMock()
-    assert validate_read_grow(AppConfig, {"blow": {"glow": 3}}) == (False, "Grow not included")
+    mock = Mock()
+    assert validate_read_grow(AppConfig, {"blow": {"glow": 3}}) == (
+        False,
+        "Grow not included",
+    )
+    mock.validate_read_grow(AppConfig, {"blow": {"glow": 3}})
```

Comment:
Same comment here, we shouldn't be calling any methods on the mock in the test (the code should call that function on the mock object)